pkg/library/libraryinputresources/types.go

```diff
@@ -16,11 +20,22 @@ type ResourceList struct {
 
 	GeneratedNameResources []GeneratedResourceID `json:"generatedNameResource,omitempty"`
 
+	LabelSelectedResources []LabelSelectedResource `json:"labelSelectedResources,omitempty"`
+
 	// use resourceReferences when one resource (apiserver.config.openshift.io/cluster) refers to another resource
 	// like a secret (.spec.servingCerts.namedCertificates[*].servingCertificates.name).
 	ResourceReference []ResourceReference `json:"resourceReferences,omitempty"`
 }
 
+type LabelSelectedResource struct {
+	InputResourceTypeIdentifier `json:",inline"`
+
+	Namespace string `json:"namespace,omitempty"`
+
+	// validation prevents setting matchExpressions
```

Comment:
validation missing in inputresources?

Reply:
Validation would be somewhere in this method: https://github.com/openshift/multi-operator-manager/blob/master/pkg/library/libraryinputresources/options.go#L42